doc/source/user_guide/troubleshoot.rst

```diff
@@ -548,4 +540,59 @@ to one, have the same shape if later retrieved.
       [3.],
       [4.]])
    >>> np.allclose(mapdl.parameters['mapdlarray'], mapdl.parameters['mapdlarray_b'])
-   True
\ No newline at end of file
+   True
+
+
+
+.. _ref_pymapdl_stability:
+
+*****************
+PyMAPDL stability
+*****************
+
+Recommendations
+===============
+
+When connecting to an instance of MAPDL using gRPC (default), there are some cases
+where the MAPDL server might exit unexpectedly. There
+are several ways to improve MADPL performance and stability:
```

Comment:
You save there are several ways but only present one. Also, you shouldn't have a list with one item.